### DISCUSSION TITLE
Can someone please send me a video tutorial on windows or hit me up on discord because I have this problem and im stuck

### DISCUSSION THREAD

**Discussion by GrimKyle** [DIGITALCRIMINALS/OnlyFans#896] [Q&A]
lol

**Comment by ne0lith** [comment]
your terminal needs to be in the same directory the start_us.py file is

**Comment by GrimKyle** [comment]
> 

can you pelase help me how do i do that

**Comment by wooki0** [comment]
> > 
> 
> can you pelase help me how do i do that

hold shift and right click and "open powershell window here"
